lib/sycamore/sycamore/transforms/partition.py

```diff
@@ -419,6 +419,10 @@ class ArynPartitioner(Partitioner):
              either pdfminer or OCR. Currently supports the 'object_type' property for pdfminer,
              which can be set to 'boxes' or 'lines' to control the granularity of output.
         source: The application that is using the partitioner. This is used for logging purposes.
+        output_label_options: A dictionary for configuring output label behavior. It supports two options: 
+        promote_title, a boolean that specifies whether to add a title to partitioned elements if one is missing, and
+            title_candidate_elements, a list of strings representing labels for potential titles.
+            default: {"promote_title": True ,  "title_candidate_elements":["Section-header", "Caption"]}
```

Comment:
I'm not sure if it makes sense to have `title_candidate_elements` as a top-level attribute given that we only care about it if `promote_title` is true but this is similar to `use_ocr` and `ocr_images` so it's not worth blocking on this.